Cargo.lock

```diff
@@ -2226,7 +2226,7 @@ dependencies = [
 [[package]]
 name = "frame-support-procedural-tools-derive"
 version = "3.0.0"
-source = "git+https://github.com/paritytech/substrate?branch=master#a6da808575fe403ab2bd7f6cd009896cdc6fd71a"
+source = "git+https://github.com/paritytech/substrate?branch=na-prep-release-sp-core-v7.0.0#ae72ad9ae66bb5d6f47e30eb79e0a81c398c0d68"
```

Comment:
The merge bot will overwrite this with the `master` branch, right?

Reply:
I think so but I can overwrite with master to be on the safe side